healthcare/.eslintrc.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
How do you feel about having these `.eslintrc.yml` and `prettierrc` files in the root of every sample?  Should we consider just having a single config at the root of the repo, and then having overrides in the individual dirs if we need em?

Reply:
I have one file at the root and overrides where needed.